joomla/joomla-cms
# [com_associations] Display Associations tab in Target

Pull Request for Issue #13812 Issue 1

### Summary of Changes
Displays now the Associations tab in Target, but still preventing modifications of existing associations in that frame.
### Testing Instructions
Install multilang site, associate some articles, load Multingual Associations, click on an article for a specific language.

You should now get this:
![screen shot 2017-02-02 at 12 28 50](https://cloud.githubusercontent.com/assets/869724/22548518/007a117e-e947-11e6-8b32-2c8c427fc068.png)

@alikon @AlexRed @rdeutz
